### ISSUE TITLE
Error Pulling ACS 2019 1-Year Data

### ISSUE THREAD

**Issue by pawbu79** [GL-Li/totalcensus#18]
Hello,

I have been a regular user of this package for the last year, mostly using it for pulling ACS 2018 5-Year data and 2010 Decennial data - and I have also pulled ACS 2018 1-Year data.

However, I am receiving an error attempting to pull the 2019 data. After setting the path, I run the follow code:

ri_state_population <- read_acs1year(
  year = 2019,
  states = c("RI"),
  table_contents = c("B01001_001"),
  summary_level = "state"
)

and I receive the following error:

Error in get(paste0("lookup_acs1year_", year)) : 
  object 'lookup_acs1year_2019' not found

Has anyone else experience this? The code snippet works just fine if I pull ACS 2018 1-Year data, and I didn't see anything in the documentation of the 2019 ACS 1-Year that indicates the field in question ("B01001_001") has changed.

Advise?

**Comment by GL-Li** [comment]
I ran your code and it worked fine.

The 2019 ACS 1-year data was added 2 months ago. You need the latest version of totalcensus, v0.6.4.

**Comment by pawbu79** [comment]
Ah, I see. Thank you for the your response.

For some reason, probably due to my install of R (3.62 currently) and RStudio (1.2.5033), the:

update.packages("totalcensus")

was not working (nothing was returned, no error, no warning). 

I checked the version, and as you indicated, I was running an older version (0.6.3). 

I wasn't able to update automatically, but after removing the package and reinstalling, I was able to run v0.6.4 and pull data from the ACS 2019 1-Year.

Thanks again!
